providers/mlx5/man/mlx5dv_create_flow_matcher.3.md

```diff
@@ -33,13 +33,19 @@ Please see *ibv_open_device(3)* for *context*.
 ## *attr*
 
 ```c
+
+enum mlx5dv_flow_matcher_attr_mask {
+	MLX5DV_FLOW_MATCHER_MASK_FT_TYPE = 1 << 0,
+};
+
 struct mlx5dv_flow_matcher_attr {
 	enum ibv_flow_attr_type type;
 	uint32_t flags; /* From enum ibv_flow_flags */
 	uint16_t priority;
 	uint8_t match_criteria_enable; /* Device spec format */
 	struct mlx5dv_flow_match_parameters *match_mask;
-	uint64_t comp_mask;
+	uint64_t comp_mask; /* use mlx5dv_flow_matcher_attr_mask */
+	enum mlx5dv_flow_table_type ft_type;
```

Comment:
And this nonsense with comments after structs is not how we write man pages now, send a fix please.

Reply:
man page notes were handled as part of updating the series with the accepted kernel headers.